src/libmongoc/src/mongoc/mongoc-client-private.h

```diff
@@ -122,6 +122,9 @@ struct _mongoc_client_t {
    mongoc_set_t *client_sessions;
    unsigned int csid_rand_seed;
 
+   int64_t timeout_ms;
+   bool timeout_set;
```

Comment:
Nit suggestion: `is_timeout_set`. Elsewhere in the driver, `is_` is commonly used as a field name prefix for bools.

Reply:
Done.